node/sc/bridge_accounts.go

```diff
@@ -35,11 +35,8 @@ import (
 	"github.com/klaytn/klaytn/params"
 )
 
-const (
-	DefaultBridgeTxGasLimit = 5000000
-)
-
 var (
+	DefaultBridgeTxGasLimit   = uint64(5000000)
```

Comment:
@hqjang-pepper Seems like this value is always reset when the process is restarted.
Could you please also make this as a parameter?
You can refer to this PR: https://github.com/klaytn/klaytn/pull/1284

Reply:
I was concerned about that issue too.
I'll add a parameter to conf file.